docs/Development-Guides/Field-Support/Compatibility.md

```diff
@@ -1,4 +1,8 @@
-# Compatibility
+---
+stoplight-id: 0517ab806dda4
```

Comment:
What does this do?

Reply:
i have no idea if this will auto-resolve the url if the path changes, but we will find out. if there's an issue after merge i'll revert